### PR TITLE
Feat : Navigation for desktop (simple).

### PR DESCRIPTION
**Pull request by shuklahimanshu81**
Screenshot (341)

### CODE REVIEW

File: components/Navigation/Navigation.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    margin:-0.75rem;
+    color: var(--black);
+    background-color: var(--gray);
+ }
+ .favourites{
+    display: flex;
+}
+.fav{
+    margin-left: -1.25rem;
+    z-index: 1;
```
Comment (FarhanMobashir) [inline components/Navigation/Navigation.css]: why z-index for fav ?

**Review by FarhanMobashir — CHANGES_REQUESTED** [review]
go through the comment

File: components/Navigation/Navigation.css (hunk 1 above)
Comment (shuklahimanshu81) [reply components/Navigation/Navigation.css]: It was going below fav icon without z-index.

Comment (FarhanMobashir) [reply components/Navigation/Navigation.css]: Alright. I think you can use the badge fro the component library that you've made. That will be quite easy

Comment (shuklahimanshu81) [reply components/Navigation/Navigation.css]: Thanks for pointing out, made the changes . Have a look again pls.

**Review by FarhanMobashir — APPROVED** [review]
All good ✅ now

**Review by bhanuprakashj — COMMENTED** [review]
looks good to me!!

**Pull request merged**